packages/playground-ui/src/index.css

```
@@ -343,8 +343,11 @@ html.light {
   --shadow-inner: inset 0 2px 4px oklch(0 0 0 / 0.3);
   --shadow-card: 0 2px 8px oklch(0 0 0 / 0.4);
```

Stylelint (17.10.0)

[error] Expected "0.4" to be "40%" (alpha-value-notation)

(alpha-value-notation)